src/Test/Perf/infra/install_vsixes.csx

```diff
@@ -25,6 +25,10 @@ var vsixes = new[]
 
 var installer = Path.Combine(binariesDirectory, "VSIXExpInstaller.exe");
 
+// Uninstall all previously installed VSIXes
+TestUtilities.ShellOutVital(installer, $"/rootSuffix:RoslynPerf /uninstallAll", IsVerbose(), logger, binariesDirectory);
```

Comment:
consider: Extract "RoslynPerf" out into a variable (and/or accept it as a command line parameter to the script).